scripts/ci/e2e-tests

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Non-blocking, but we need to figure out the purpose of e2e tests for a CLI. It's not clear to me these are particularly useful. The deployment of the engine etc is not part of the CLI test since the engine image isn't part of the release process for the CLI. Functional tests are probably sufficient for the CLI.

Reply:
I agree. I plan to review e2e/functional/integration tests across projects to ask questions in this context.

Reply:
The e2e-tests (for all projects) are more of an image smoke test then a true e2e test at this point. If we remove e2e tests from the cli project, there will be no smoke tests of the actual built image. We could do something less complicated, but I like the idea of all our projects having a similar CI flow: lint -> unit/integration/functional tests -> build -> e2e -> push -> image-scan